### DISCUSSION TITLE
Remove some role permissions

### DISCUSSION THREAD

**Discussion by iamshoXy** [WeblateOrg/weblate#11442] [Q&A] [answered]
How would I change role permissions? For example removing translation.add for the 'Translate' role?

I tried removing it in `weblate/auth/data.py`, more specifically in the TRANSLATE_PERMS Set

**Comment by nijel** [comment, marked as answer]
The built-in roles can not be changed, see https://docs.weblate.org/en/latest/admin/access.html#default-teams-and-roles
